scripts/extensions/auto-tagging-widget/src/ImageTaggingComponent/ImageTaggingComponent.tsx

```diff
@@ -0,0 +1,278 @@
+import * as React from "react";
+import { ISuperdesk } from "superdesk-api";
+import { ToggleBoxNext } from "superdesk-ui-framework";
+
+interface ITag {
+  title: string;
+  type: string;
+  pubStatus: boolean;
+  weight: number;
+}
+
+interface IImage {
+  imageUrl: string;
+  thumbnailUrl: string;
+  id?: string;
+  headline?: string;
+  caption?: string;
+  credit?: string;
+  byline?: string;
+  source?: string;
+  dateCreated?: string;
+  archivedTime?: string;
+}
+
+interface IProps {
+  superdesk: ISuperdesk;
+  data: any;
```

Comment:
Do not use `any`. Specify the actual type. Here and in other places in this PR.

Reply:
This is now solved. IServerResponse from the adapter is now used. Thanks once again for the feedback! :)